### ISSUE TITLE
"Send Request" button is not accessible using the keyboard: A11y_VS Code extension for API Center_Generate HTTP file_Send Request_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#99]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG2.1.1; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #A11yMAS;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Generate HTTP file" using the down arrow and press enter.
7. An HTTP file will appear.
8. Verify whether the "Send Request" button is accessible using the keyboard or not.

### Actual Result:
"Send Request" button is not accessible using the keyboard when we are navigating using the tab key.

### Expected Result:
"Send Request" button should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the button.

### Attachment:
![Bug_99](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/8894aba7-e3ca-4514-b12c-7792d18d7a5d)

**Comment by formulahendry** [comment]
External issue. This UI control is from REST Client extension: https://github.com/Huachao/vscode-restclient

**Comment by formulahendry** [comment]
This issue is caused by UI of REST Client extension, not related to UI of API Center extension.

**Comment by AnKushSingh05** [comment]
IRT Bug Logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/223550
